routers/api/v1/misc/markdown.go

```diff
@@ -61,7 +63,14 @@ func Markdown(ctx *context.APIContext, form api.MarkdownOption) {
 			}
 		}
 		if ctx.Repo != nil && ctx.Repo.Repository != nil {
-			meta = ctx.Repo.Repository.ComposeMetas()
+			if form.Mode == "gfm" {
```

Comment:
```suggestion
			// "gfm" = GitHub Flavored Markdown, as used by the javascript library
			if form.Mode == "gfm" {
```
Replace "the javascript library" with the library name 😁

Reply:
Library? It's what's the API uses...

```suggestion
			// "gfm" = Github Flavored Markdown.
			if form.Mode == "gfm" {
```

Reply:
OK, it was confusing having two different names for the same setting ("gfm" vs "document"). But adding the comment is good enough. 👍